### PR TITLE
Feature#182201735 travel request updates notifications

### PR DESCRIPTION
**Pull request by Patricknshimiyimana**
to create a socket connection
> - Send a request like create trip request on the api
> - Go back on the static page and you will see notifications already in the console

### CODE REVIEW

File: socketClient/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```
Comment (joshwambere) [inline socketClient/index.html:1]: at least use few styling

File: socketClient/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      e.preventDefault();
+      const token = document.querySelector('.token').value;
+
+      const socket = io({
+        auth: {
+          token,
+        },
+      });
+      socket.on('newNotification', (msg) => {
+        console.log(msg);
```
Comment (joshwambere) [inline socketClient/index.html]: create a good looking UI and display those notifications not only in the console

**Review by joshwambere — CHANGES_REQUESTED** [review]

File: socketClient/index.html (hunk 1 above)
Comment (Patricknshimiyimana) [reply socketClient/index.html:1]: Done!

File: socketClient/index.html (hunk 2 above)
Comment (Patricknshimiyimana) [reply socketClient/index.html]: Done!
I Think the other work will be done on the frontend
Screen Shot 2022-07-25 at 22 24 15

File: test/socket.io.test.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { createServer } from 'http';
+import { io as Client } from 'socket.io-client';
+import { Server } from 'socket.io';
+import { assert } from 'chai';
+
+// with { "type": "commonjs" } in your package.json
```
Comment (joshwambere) [inline test/socket.io.test.js]: remove those comments

**Review by joshwambere — CHANGES_REQUESTED** [review]

File: test/socket.io.test.js (hunk 3 above)
Comment (Patricknshimiyimana) [reply test/socket.io.test.js]: Done.
Thanks!

**Review by joshwambere — APPROVED** [review]

**Pull request merged**